src/main/app/fees/feesClient.ts

```diff
@@ -1,18 +1,19 @@
 import * as config from 'config'
 import { request } from 'client/request'
 import { plainToClass } from 'class-transformer'
-
-import { CalculationOutcome } from 'app/fees/models/calculationOutcome'
-import { RangeGroup } from 'app/fees/models/rangeGroup'
-
 import { ClaimValidator } from 'app/utils/claimValidator'
+import { FeeOutcome } from 'fees/models/feeOutcome'
+import { FeeRange } from 'fees/models/feeRange'
 import { StringUtils } from 'utils/stringUtils'
-import { MoneyConverter } from 'app/fees/moneyConverter'
 
 const feesUrl = config.get('fees.url')
-
-const issueFeeCode = config.get<string>('fees.issueFee.code')
-const hearingFeeCode = config.get<string>('fees.hearingFee.code')
+const service: string = config.get<string>('fees.service')
+const jurisdiction1: string = config.get<string>('fees.jurisdiction1')
+const jurisdiction2: string = config.get<string>('fees.jurisdiction2')
+const onlineChannel: string = config.get<string>('fees.channel.online')
+const defaultChannel: string = config.get<string>('fees.channel.paper')
```

Comment:
I would consider renaming it to `paperChannel` so that configuration and TS code use the same language. I like the online vs paper approach.